assets/components/directDebit/directDebitActions.js

```diff
@@ -50,12 +52,13 @@ function payDirectDebitClicked(callback: Function): Function {
   return (dispatch: Function, getState: Function) => {
 
     const {
-      bankSortCode,
+      sortCode,
       bankAccountNumber,
       accountHolderName,
       accountHolderConfirmation,
     } = getState().page.directDebit;
 
+    const bankSortCode = sortCode.join('');
```

Comment:
This feels a little hacky, I would rename `bankSortCode` here to be `candidateSortCode`.

Reply:
I've renamed where it is stored as an array to sortCodeArray and this to sortCode. I think this is clearer on both counts.